### ISSUE TITLE
Vanilla JS PATCH request

### ISSUE THREAD

**Issue by gdibble** [winduptoy/ampersand-json-patch#3]
why send all of zepto up to the client when this can be converted to a Vanilla JS PATCH?
https://github.com/winduptoy/ampersand-json-patch/blob/master/index.js#L116

**Comment by gdibble** [comment]
concern for users who do not use zepto but care about overall app weight

**Comment by gdibble** [comment]
good refs http://stackoverflow.com/questions/8567114/how-to-make-an-ajax-call-without-jquery

**Comment by winduptoy** [comment]
@gdibble Yeah I was planning to get to it, but I never found the time. If you submit a PR I'll be happy to merge it.

**Comment by gdibble** [comment]
ok, i'll see what i can do ;)
